### ISSUE TITLE
Plantuml server is not working on Tomcat 10

### ISSUE THREAD

**Issue by mkde1** [plantuml/plantuml-server#237]
I followed the instructions under the title of "Installation of the PlantUML Server" and installed "Tomcat 10" as the JEE 5 web server according to the link below. Then I downloaded the "plantuml.war" file and placed it inside "C:\Program Files\Apache Software Foundation\Tomcat 10.0\webapps\". After placing "plantuml.war" file, "plantuml" directory has been created automatically. 

Tomcat 10 server works without a problem but when I hit "localhost:8080/plantuml" it gives 404 error.
link: https://plantuml.com/server

Two screenshots have been added.
![webappmanager](https://user-images.githubusercontent.com/69921915/182859308-d97e30cf-4bcc-4e8a-9a3d-5dbc174a05dd.png)
![plantuml-404](https://user-images.githubusercontent.com/69921915/182859357-d6831395-c65b-4252-990d-9e2eb1f87e97.png)

**Comment by mkde1** [comment]
**UPDATE**
According to localhost log file I've encountered with this error
SEVERE [http-nio-8081-exec-1] org.apache.catalina.core.StandardWrapperValve.invoke Allocate exception for servlet [plantumlservlet]
	java.lang.UnsupportedClassVersionError: net/sourceforge/plantuml/servlet/PlantUmlServlet has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0 (unable to load class [net.sourceforge.plantuml.servlet.PlantUmlServlet])

**Comment by mkde1** [comment]
**SECOND UPDATE** --ISSUE RESOLVED--
1-I have switched to JDK instead of JRE. 
2-Then I started to receive jsp error(503) instead of runtime error I was taking before.
`SEVERE [http-nio-8080-exec-10] org.apache.catalina.core.StandardContext.loadOnStartup Servlet [jsp] in web application [/plantuml] threw load() exception
	java.lang.ClassNotFoundException: org.eclipse.jetty.jsp.JettyJspServlet`
3-Fortunately, I found this link: https://github.com/plantuml/plantuml-server/releases
4-Downloaded "plantuml-jsp-v1.2022.6.war" and deployed it.
5-It worked.

SUGGESTION: If you somehow download from this source "https://sourceforge.net/projects/plantuml/files/plantuml.war/download" and doesn't work(404 or 503 error), then try downloading files from this address "https://github.com/plantuml/plantuml-server/releases".
